### DISCUSSION TITLE
Is this clone connected to Twitter in real time?

### DISCUSSION THREAD

**Discussion by nniiggeell** [TheAlphamerc/flutter_twitter_clone#199] [Q&A] [answered]
Hi, the app works great. But is there a way for the app to be connected to Twitter in real time. Am I able to log in to my actual Twitter account and view tweets that from people I actually follow in my real Twitter app?

If so, how can I do it?

**Comment by TheAlphamerc** [comment, marked as answer]
Hi @nniiggeell, Fwitter is a clone of Twitter and has its own backend from where all data is fetched and stored. The Fwitter doesn't have the functionality to login into the app using Twitter credentials neither can fetch tweets from it.
